### DISCUSSION TITLE
Text stroke on after element generates a function

### DISCUSSION THREAD

**Discussion by Ashenbacken** [tailwindlabs/tailwindcss#13101] [Help] [answered]
Using the following class in JSX generates a function value in the CSS output.

Class used: `after:text-stroke-yellow-300`

Generated output:
```
.after\:text-stroke-yellow-300::after {
	content: var(--tw-content);
	-webkit-text-stroke-color: ({ opacityVariable, opacityValue }) => {
        if (!isNaN(+opacityValue)) {
          return `hsl(var(${twcColorVariable}) / ${opacityValue})`;
        }
        if (opacityVariable) {
          return `hsl(var(${twcColorVariable}) / var(${twcOpacityVariable}, var(${opacityVariable})))`;
        }
        return `hsl(var(${twcColorVariable}) / var(${twcOpacityVariable}, 1))`;
      };
}
```

**Comment by wongjn** [comment, marked as answer]
I believe this `text-stroke-*` utility is from a Tailwind plugin not included with core Tailwind. Consider raising this issue with the developers of this third-party plugin, or if it is your own plugin consider providing the code of the your plugin.

**Reply by Ashenbacken** [reply]
Yes your are right, thank you for the quick reply!
